Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine3.12
+
+#install asp.net libraries
+RUN aspnetcore_version=3.1.11 \
```

Comment:
Doesn't sdk come with the dotnet cli?

Reply:
Yes sdk comes with dotnet CLI.

Reply:
Then why do you have to install aspnetcore?